examples/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -138,3 +139,60 @@ objects:
   - kind: ServiceAccount
     namespace: ${NAMESPACE}
     name: apiserver
+
+# for the SA for the service catalog to use when calling the tsb
```

Comment:
nit, but these comments didn't help this reviewer.  Suggested alternatives:
# this SA will be able to call the TSB API
# actually permit the above SA to call the TSB API
# (please explain what this does and why it's here)
# permit kube-service-catalog/service-catalog-controller to read this secret

Reply:
comments added, but this PR is still going to need to rebased on top of @deads2k's file movements before it can be merged.